ProjWidget.py

```diff
@@ -154,7 +328,69 @@ def makeImage(self, xminR, xmaxR, yminR, ymaxR):
         # At this point, we should plot whatever we want to plot and fit whatever
         # we want to fit.
 
-        ax.plot(x, y, "g-")
+        self.yplot = None
+        if self.gui.ui.checkBoxM1Lineout.isChecked():
+            (ymin, ymax) = self.plotLineout(
+                ax,
+                self.is_x,
+                linelim,
+                x,
+                idx,
+                xmin,
+                xmax,
+                ymin,
+                ymax,
+                self.gui.ui.display_image.lMarker[0].oriented(),
+                self.gui.ui.display_image.lPenColor[0],
+            )
+        if self.gui.ui.checkBoxM2Lineout.isChecked():
+            (ymin, ymax) = self.plotLineout(
+                ax,
+                self.is_x,
+                linelim,
+                x,
+                idx,
+                xmin,
+                xmax,
+                ymin,
+                ymax,
+                self.gui.ui.display_image.lMarker[1].oriented(),
+                self.gui.ui.display_image.lPenColor[1],
+            )
+        if self.gui.ui.checkBoxM3Lineout.isChecked():
+            (ymin, ymax) = self.plotLineout(
+                ax,
+                self.is_x,
+                linelim,
+                x,
+                idx,
+                xmin,
+                xmax,
+                ymin,
+                ymax,
+                self.gui.ui.display_image.lMarker[2].oriented(),
+                self.gui.ui.display_image.lPenColor[2],
+            )
+        if self.gui.ui.checkBoxM4Lineout.isChecked():
+            (ymin, ymax) = self.plotLineout(
+                ax,
+                self.is_x,
+                linelim,
+                x,
+                xmin,
```

Comment:
Is `idx` a missing parameter here by chance? 

My eyes were drawn to the 10ish positional parameters and this seemed to not fit the pattern of the above, but I can't really tell. Might consider a loop to get rid of the copy/paste, if so

Reply:
Absolutely correct.  I've modified this in another commit.  Meanwhile, found another bug... my code for deciding whether or not to display a lineout was looking at the wrong dimension.

It's always something...